### DISCUSSION TITLE
SLO with Okta

### DISCUSSION THREAD

**Discussion by lzaiko** [bergie/passport-saml#604] [Q&A]
Hi guys
I'm trying to implement the SLO (Single Lougout) on Okta  but i'm missing a `Singning Certificate` apparently.

The issue is described [here](https://support.okta.com/help/s/question/0D54z00006z0onUCAQ/how-to-correctly-perform-a-slo-from-service-provider?language=en_US).

Is there any examples explaining the steps (how to create the certificate and how to use it in passport)?
Couldn't find anything in the test directory.
Thanks
Louis

**Comment by markstos** [comment]
Be aware that there's a known issue with our SLO implementation: https://github.com/node-saml/passport-saml/issues/419
